jest.config.js

```diff
@@ -0,0 +1,54 @@
+var path = require('path');
```

Comment:
Is there anything prohibiting us from favoring `const` over `var` where it is necessary?